config/boards/cherryba-m1.csc

```diff
@@ -0,0 +1,11 @@
+ # Allwinner H618 quad core 1GB 2GB 4GB RAM SoC WiFi USB-C emmc
+BOARD_NAME="CherryBa M1"
+BOARDFAMILY="sun50iw9"
```

Comment:
add:
BOARD_MAINTAINER="IsMrX"
in case you would like to be maintainer. Doesn't have affect [on support status](https://docs.armbian.com/User-Guide_Board-Support-Rules/) and leave [additional contacts](https://www.armbian.com/update-data/) if you want.

Reply:
OK。It has been added